src/bp/core/config/bot.config.ts

```diff
@@ -35,16 +35,10 @@ export interface BotPipelineStatus {
     expires_on?: Date
     requested_by: string
     id: string
-    approvers?: StageRequestApprovers[]
+    approvals?: string[]
```

Comment:
A UniqueUser is represented by the strategy + email. I suggest keeping the original name & object, maybe just remove "approved" from the prop list